steps/appgallery-deploy/assets/icon.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 256 256" style="enable-background:new 0 0 256 256;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill-rule:evenodd;clip-rule:evenodd;fill:#C8102E;}
+	.st1{fill:#FFFFFF;}
+</style>
+<g>
+	<g id="形状_1_拷贝_25_9_">

```

Comment:
It looks weird to have non-ASCII id.
Is it really intended?

Reply:
It should be alright, the svg has been generated from Chinese resources within my team. Does an ID in a .svg generate any issue? @koral--

Reply:
probably it wouldn't cause any issue but I have removed the id just in case